mix/MixClient.cpp

```diff
@@ -44,27 +44,18 @@ namespace mix
 
 const Secret c_defaultUserAccountSecret = Secret("cb73d9408c4720e230387d956eb0f829d8a4dd2c1055f96257167e14e7169074");
 const u256 c_mixGenesisDifficulty = c_minimumDifficulty; //TODO: make it lower for Mix somehow
-
-class MixBlockChain: public dev::eth::BlockChain
+	
+bytes MixBlockChain::createGenesisBlock(h256 _stateRoot)
```

Comment:
Actually not, it was not a part of this pull request. It is actually result of my incorrect merge with develop branch. I haven't changed this. Thanks for careful review. I will fix it.